crates/dpapi-native-transport/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "dpapi-native-transport"
+version = "0.1.0"
+edition = "2024"
```

Comment:
suggestion: Can you add the required meta data needed for publishing crates?

Reply:
done (sorry for the delay, I had a meeting)

Reply:
No problem! Thank you!